tests/trait_impl.rs

```diff
@@ -6,6 +6,8 @@ extern crate static_assertions;
 
 use core::ops::Range;
 
+const_assert!(true);
```

Comment:
Wrong test file for this assertion. I'm guessing this was committed by accident?

Reply:
Yes sorry